pkgs/by-name/ja/jailer/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  copyDesktopItems,
```

Comment:
How are these things sorted? Anyway we generally put lib first.

Reply:
Done, I put lib first, then alphabetical order